client/src/js/wall/Login.js

```diff
@@ -69,21 +70,20 @@ export class Login extends React.Component {
                 <WallLogo height={42} />
                 <LoginModal>
                     <form onSubmit={this.handleSubmit}>
-                        <Input
-                            type="text"
-                            label="Username"
-                            name="username"
-                            value={this.state.username}
-                            onChange={this.handleChange}
-                            autofocus
-                        />
-                        <Input
-                            type="password"
-                            label="Password"
-                            name="password"
-                            value={this.state.password}
-                            onChange={this.handleChange}
-                        />
+                        <InputGroup>
+                            <InputLabel>Username</InputLabel>
+                            <Input
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Replace `⏎································name="username"⏎································value={this.state.username}⏎································onChange={this.handleChange}⏎································autofocus⏎···························` with `·name="username"·value={this.state.username}·onChange={this.handleChange}·autofocus`](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5211749)